acceptance/tests/direct_puppet/supports_utf8.rb

```diff
@@ -1,6 +1,10 @@
 test_name "C97172: static catalogs support utf8" do
-require 'puppet/acceptance/environment_utils'
-extend Puppet::Acceptance::EnvironmentUtils
+
+  confine :except, :platform => 'cisco_' # skip the test because some of the machines have LANG=C as the default and we break
```

Comment:
Should this be `/^cisco_/` instead of `'cisco_'`? Every other instance of the partial platform name I see in the acceptance tests uses the regex form.

Reply:
Sure